bsp/esp32_s3_korvo_1/esp32_s3_korvo_1.c

```diff
@@ -0,0 +1,157 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "esp_err.h"
+#include "esp_log.h"
+#include "led_strip.h"
+
+#include "iot_button.h"
+#include "bsp/esp-bsp.h"
+#include "bsp_err_check.h"
+
+static const char *TAG = "ESP32-S3-Korvo-1";
+
+static led_strip_handle_t led_strip;
+
+/**
+ * @brief led configuration structure
+ *
+ * This configuration is used by default in bsp_led_init()
+ */
+static const led_strip_config_t bsp_strip_config = {
+    .strip_gpio_num = BSP_RGB_CTRL,
+    .max_leds = BSP_RGB_LEDS_NUM,
+    .led_pixel_format = LED_PIXEL_FORMAT_GRB,
+    .led_model = LED_MODEL_WS2812,
+    .flags.invert_out = false,
+};
+
+static const led_strip_rmt_config_t bsp_rmt_config = {
+    .clk_src = RMT_CLK_SRC_DEFAULT,
+    .resolution_hz = 10 * 1000 * 1000,
+    .flags.with_dma = false,
```

Comment:
`clk_src`, `resolution_hz`, `flags.with_dma` - not supported in IDF4.4, we should solve it. And `channel` is in IDF4.4 structure. For example: https://github.com/espressif/esp-bsp/blob/51c8afb977b169dbcb107dcbd2d6120a4926fd54/bsp/esp_bsp_generic/src/esp_bsp_generic.c#L164